CODING_STANDARDS.MD

```diff
@@ -0,0 +1,24 @@
+# Coding Standards 
+
+### General Best Practices
+- Make sure to add unit test cases when adding new functionality and ensure added code is written in a testable manner.
+- All functions should have a description of their purpose and details of the parameters.
+- Soft limit of 100 characters per line.
+- Each issue should have one pull request unless the changes are only 1-3 lines.
+
+### Python Best Practices
+- Use snake-case for multi-word classes, variables, and functions.
+
+### JavaScript Best Practices
```

Comment:
are these team prefs, or based on any standard styles? just curious.

Reply:
For Python, we gave this as per the PEP variable name standard (https://peps.python.org/pep-0008/#function-and-variable-names) and for react js, we referred to a few sites for clean coding and coding standards. This is also as per current project standards.